lib/pynml/nml.py

```diff
@@ -2447,7 +2447,7 @@ def get_has_link(self):
         return copy(self._has_link_links)
 
 
-class Environment(NMLObject):
+class Environment(NetworkObject):
```

Comment:
The reason for this fix is that  [ExtendedNMLManager.register_object](https://github.com/HPENetworking/pynml/blob/master/lib/pynml/manager.py#L103) expects the object to be a instance of NetworkObject.
Since NMLObject does not have the identifier attribute, it complains

Reply:
```
______________________ ERROR at setup of test_lonely_node ______________________
Error building topology in module test_topology_lonely_node:
Traceback (most recent call last):
  File "/home/travis/build/HPENetworking/topology/.tox/py3/lib/python3.4/site-packages/topology/pytest/plugin.py", line 167, in topology
    topomgr.parse(topo, inject=suite_injected_attr)
  File "/home/travis/build/HPENetworking/topology/.tox/py3/lib/python3.4/site-packages/topology/manager.py", line 206, in parse
    self.load(data, inject=inject)
  File "/home/travis/build/HPENetworking/topology/.tox/py3/lib/python3.4/site-packages/topology/manager.py", line 131, in load
    self.nml.create_environment(**environment)
  File "/home/travis/build/HPENetworking/topology/.tox/py3/lib/python3.4/site-packages/pynml/manager.py", line 334, in create_environment
    self.register_object(environment)
  File "/home/travis/build/HPENetworking/topology/.tox/py3/lib/python3.4/site-packages/pynml/manager.py", line 110, in register_object
    if obj.identifier in self.namespace:
AttributeError: 'Environment' object has no attribute 'identifier'
```